diff --git i/main.go w/main.go
index 0a85b02f..a16a98fd 100644
--- i/main.go
+++ w/main.go
@@ -3,6 +3,7 @@ package main
 import (
 	_ "embed"
 	"fmt"
+  "syscall"
 	"os"
 	"os/exec"
 	"strings"
@@ -41,8 +42,10 @@ func printScript(label string, content []byte) {
 func exit(code int, err error) {
 	if code == fzf.ExitError && err != nil {
 		fmt.Fprintln(os.Stderr, err.Error())
-	}
-	os.Exit(code)
+    syscall.Exec("/bin/false", []string{"/bin/false"}, os.Environ())
+	} else {
+   syscall.Exec("/bin/true", []string{"/bin/true"}, os.Environ())
+}
 }
 
 func main() {
